packages/module/src/LogViewer/LogViewer.tsx

```diff
@@ -113,13 +115,21 @@ const LogViewerBase: React.FunctionComponent<LogViewerProps> = memo(
     let resizeTimer = null as any;
 
     useEffect(() => {
+      let observer : ResizeObserver | undefined = undefined;
       if (containerRef && containerRef.current) {
         window.addEventListener('resize', callbackResize);
+        observer = new ResizeObserver((event) => {
+          setEmptyState({});
```

Comment:
I tried calling `callbackResize` here put it interfered with scrolling behavior (resize is called even for scrolling).

Reply:
[curious-question]: Are we using `setEmptyState` as a way to rerender the component, could be possible to use `setLoading(false)` as well? would that work?

Reply:
`setLoading(false)`: Same behavior as before the fix 
`setLoading(true)` or `setResizing(true)`: the log container is empty